client/src/App.js

```diff
@@ -15,6 +16,7 @@ function App() {
         <Switch>
           <Route exact path="/" component={LandingPage} />
           <Route path="/login" component={Login} />
+          <Route path="/createcontest" component={CreateContest} />
```

Comment:
i think @maziasy 's route naming convention was in snake case like `view-submission`. this should either be `createContest` or `create-contest` to stay consistent :)

Reply:
changed it to `/create-contest` for consistency!